src/views/registration-js/attendees/fields/email.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/email.php
+ *
+ * @since  TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Email
+ */
+
+$required   = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id  = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$field      = (array) $field;
+$field_name = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled   = false;
+$classes    = [
+		'tribe-common-b1',
+		'tribe-field',
+		'tribe-tickets__item__attendee__field__email',
+		'tribe-tickets-meta-required' => $required
+];
```

Comment:
* :no_entry_sign: Array item not aligned correctly; expected 4 spaces but found 8
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: 50812a25fb3ce246b804a9475914b9ca</sub>